src/pages/Vote.js

```diff
@@ -11,11 +11,15 @@ function Vote() {
   const history = useHistory();
   const [poll, setPoll] = React.useState(null);
   const [answer, setAnswer] = React.useState(null);
+  const [isLoadingPatchPoll, setIsLoadingPatchPoll] = React.useState(false);
+  const [isLoadingGetPoll, setIsLoadingGetPoll] = React.useState(true);
 
   React.useEffect(() => {
     async function doGetPoll() {
+      setIsLoadingGetPoll(true);
```

Comment:
Why you set ```setIsLoadingGetPoll(true);``` `true`?
The initial state of isLoadingGetPoll is `true` already, look line 15